src/pyodide/internal/metadata.ts

```diff
@@ -52,7 +54,8 @@ export interface CloudflareGlobal {
   };
 }
 
-const compatibilityFlags: CompatibilityFlags =
+// WARNING: this doesn't seem to support experimental flags.
```

Comment:
That's a bit strange. Is there some code somewhere that explicitly makes sure experimental flags aren't exposed?

Reply:
Yeah, it is strange.

Looking into it a bit more... it looks like https://github.com/cloudflare/workerd/blob/main/src/workerd/api/global-scope.c%2B%2B#L1004 this is the code and it only exposes the experimental flags if experimental mode is enabled. So even if you enable an experimental flag, you won't see it in this structure.